src/main/java/paqua/loan/amortization/dto/Loan.java

```diff
@@ -164,17 +164,38 @@ public LoanBuilder amount(BigDecimal amount) {
             return this;
         }
 
-        /**
+		/**
```

Comment:
It seems that the indent is bigger than the rest of the code in the builder class. It would be great if you can align the new code like the rest of the code.